src/components/global/Column.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+// import { Link } from "react-router-dom";
```

Comment:
Some commented out code of unused imports. Should be removed if they wont be used

Reply:
Agreed